src/components/main-form.js

```diff
@@ -80,22 +81,44 @@ const AmortizationTable = styled(Table)`
     border-left-color: ${GRAY}};
   }
 `
+const emphasize = cx('card', css(`border-color: ${LIGHT_GREEN};`))
+
+const box = bestOption => css`
+  background-color: ${bestOption ? LIGHT_GREEN : GRAY};
```

Comment:
Since we already have an `<something>Option` pattern, perhaps this should be `isBestOption` to be a boolean?

Reply:
that's better, I'll correct it